docker/scripts/install-docker-compose-ubuntu.sh

```diff
@@ -0,0 +1,28 @@
+# Ask for the user password
```

Comment:
It's usually better to start bash scripts at least with `set -e` flag (ideally with the [famous `set -euxo pipefail` ones](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/)). Bash is pretty permissive when it comes to failures and unsafe behaviours:

https://github.com/rhcs-dashboard/ceph-dev/blob/3aa188a5ed2efaa142b9bfe2088743a29b405e92/docker/scripts/install-docker-compose-centos-rhel.sh#L3